elixir_grpc_bench/lib/helloworld.pb.ex

```diff
@@ -0,0 +1,91 @@
+defmodule Helloworld.Hello.Pet.Color do
```

Comment:
can this file be also generated during build phase and not added to version control?

Reply:
@sleipnir ?

Reply:
Hi @LesnyRumcajs, I've been away for a few days, I'll take a look at this as soon as possible.